lbr_examples/CMakeLists.txt

```diff
@@ -20,9 +20,16 @@ install(
     PROGRAMS
         scripts/admittance_control_node.py
         scripts/lbr_command_node.py
+	scripts/lbr_joint_state_publisher.py
+	scripts/lbr_ext_torque_example.py
```

Comment:
this script should come in between `lbr_command_node` and `lbr_joint_state_publisher`. Please remove `lbr_joint_state_publisher` as redundant, see below